docs/src/main/asciidoc/kafka-streams-guide.adoc

```diff
@@ -22,6 +22,32 @@ To complete this guide, you need:
 
 It is recommended, that you have read the {quickstarts-tree-url}/kafka-quickstart[Kafka quickstart] before.
 
+[NOTE]
+====
+The Quarkus extension for Kafka Streams allows for very fast turnaround times during development by supporting the Quarkus Dev Mode (e.g. via `mvn compile quarkus:dev`).
+After changing the code of your Kafka Streams topology, the application will automatically be reloaded when the next input message arrives.
+
+A recommended development set-up is to have some producer which creates test messages on the processed topic(s) in fixed intervals, e.g. every second and observe the streaming application's output topic(s) using a tool such as `kafkacat`.
+Using the dev mode, you'll instantly see messages on the output topic(s) as produced by the latest version of your streaming application when saving.
+
+For the best development experience, we recommend the following configuration settings to your Kafka broker:
```

Comment:
`following` should perhaps be `applying`?

Reply:
Also do we need both `configuration` and `settings`?

Reply:
Good catch on "applying" missing. Added it. I think the other is fine, but feel free to change it if you prefere.

Reply:
Nah, just a suggestion. I don't mind either way :).

Could you please squash however so I can merge?